api/Document.json

```diff
@@ -174,16 +174,16 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "1"
```

Comment:
Based on the PR description I didn't expect to see Safari changes, and there are only a few. I'll push a commit to revert those to make it Chromium-only.